src/devices/Shtc3/samples/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Common;
+using UnitsNet;
+
+namespace Iot.Device.Shtc3.Samples
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            I2cConnectionSettings settings = new I2cConnectionSettings(1, Iot.Device.Shtc3.Shtc3.I2cAddress);
+            I2cDevice device = I2cDevice.Create(settings);
+
+            using (Shtc3 sensor = new Shtc3(device))
+            {
+                Console.WriteLine($"Sensor Id: {sensor.Id}");
+                while (true)
+                {
+                    // Set sensor ready to use because while apply sleep mode bellow
+                    sensor.Status = Status.Idle;
```

Comment:
I'd prefer the status was only exposed by method/properties rather than direct access, i.e.:

```csharp
sensor.Sleep(); // sets status to sleep
sensor.WakeUp(); // and/or call it automatically on use
// not sure what Status = Idle is actually doing...
// etc
```